pages/en/publish/play-billing-backend.md

```diff
@@ -13,29 +13,29 @@ The Google Play Developer API has two components: the [Subscriptions and In-app
 
 - [`inappproducts`](https://developers.google.com/android-publisher/api-ref/rest/v3/inappproducts): in-app products and subscriptions catalog management
 - [`purchases.products`](https://developers.google.com/android-publisher/api-ref/rest/v3/purchases.products): in-app products purchase status
-- [`purchases.subscriptions`](https://developers.google.com/android-publisher/api-ref/rest/v3/purchases.subscriptions): subscriptions purchase status
+- [`purchases.subscriptions`](https://developers.google.com/android-publisher/api-ref/rest/v3/purchases.subscriptions): subscriptions purchase status and management
 
-You can use the Google Play Developer API directly as a REST API, but the [client libraries](https://developers.google.com/android-publisher/libraries) make it easy to quickly start developing. The client libraries for all supported languages can be found [here](https://developers.google.com/api-client-library). Follow the [Getting Started guide](https://developers.google.com/android-publisher/getting_started) for the Google Play Developer API to link your API project and set up API access clients.
+You can use the Google Play Developer API directly as a REST API, but the [client libraries](https://developers.google.com/android-publisher/libraries) make it easier to quickly start developing. The client libraries for all supported languages can be found [here](https://developers.google.com/api-client-library). Follow the [Getting Started guide](https://developers.google.com/android-publisher/getting_started) for the Google Play Developer API to link your API project and set up API access clients.
```

Comment:
Can we find a different word for "easier"

Reply:
changed the wording so we're not using "easy" or "easier"